applications/Chat/evaluate/README.md

```diff
@@ -1,182 +1,217 @@
+        "input": "",
+        "output": "",
+        "target": "{target answer}",
+        "id": 2
+    }
+]
+```
+
+#### Model Answers / Predictions
+
+A JSON file contains one list. Each element in the list is a model answer / prediction record for one instruction / question.
+
+An element should have the following fields:
+
+* `category` (str, compulsory): The category of the instruction / question.
+* `instruction` (str, compulsory): The instruction / question for the LLM.
+* `input` (str, optional): The additional context of the instruction / question.
+* `output` (str, compulsory): The output from the LLM.
+* `target` (str, optional): The target answer for the instruction.
+* `id` (int, compulsory): The ID of the instruction / question.
+
+Example:
+```
+[
+    {
+        "category": "brainstorming",
+        "instruction": "请介绍一下人工智能的多个领域。",
+        "input": "",
+        "output": "{GPT-3.5 Answers}",
+        "target": "{Model Answers / Predictions}",
+        "id": 1
+    },
+    {
+        "category": "classification",
+        "instruction": "新闻标题：为什么电影《倩女幽魂》中燕赤霞一个道士却拿着金刚经？请根据新闻标题判断新闻所属的分类，你需要从文化，娱乐，体育，财经，房产，教育，科技，旅游，游戏，军事这十类中选择一个答案。",
+        "input": "",
+        "output": "{Model Answers / Predictions}",
+        "target": "{target answer}",
+        "id": 2
+    }
+]
+```
+
+### Evaluation
+#### Configuration
+The configuration file `config_cn.json` can control how evaluate the performance of the model.
+The following is an example showing the config structure:
+```
+{
+    "language": "cn",
+    "category": {
+        "brainstorming": {
+            "GPT-3.5": ["relevance", "creativity", "practicality", "correctness"],
+            "GPT-4": [],
+            "Metrics": ["Distinct"]
+        },
+        "chat": {
+            "GPT-3.5": [ "relevance", "naturalness", "engagingness", "reasonableness"],
+            "GPT-4": [],
+            "Metrics": ["Distinct"]
+}
+```
+`"language"`: evaluate the model capability in which language, choosing from `"cn"` (we only support Chinese for now).
+`"category"`: evaluate the model capability in which category/categories, choosing from `"brainstorming"`,`"chat"`,`"classification"`,`"closed_qa"`,`"extraction"`,`"generation"`,`"open_qa"`,`"rewriting"`,`"roleplay"`,`"summarization"`.
+`"GPT-3.5"`: config metrics for GPT-3.5 evaluation, choosing from `"language organization"`,`"relevance"`,`"creativity"`,`"practicality"`,`"correctness"`,`"naturalness"`,`"engagingness"`,`"reasonableness"`,`"diversity"`,`"fidelity"`,`"conciseness"`.
+`"GPT-4"`: config metrics for GPT-4 evaluation, choosing from `"language organization"`,`"relevance"`,`"creativity"`,`"practicality"`,`"correctness"`,`"naturalness"`,`"engagingness"`,`"reasonableness"`,`"diversity"`,`"fidelity"`,`"conciseness"`.
+`"Metrics"`: config metrics for automatic metrics evaluation, choosing from `"BLEU"`,`"ROUGE"`,`"Distinct"`,`"BERTScore"`,`"Precision"`,`"Recall"`,`"F1 score"`.
+
```

Comment:
<img width="1141" alt="image" src="https://github.com/hpcaitech/ColossalAI/assets/52250342/0373140c-a156-4a70-8943-32aa1285281e">
Please reformat this as it looks very tight.

Reply:
@chengeharrison can you double-check the metric for "GPT-3.5"?